theoads/getting-started/00-getting-started-signaling-service.mdx

```diff
@@ -131,7 +131,8 @@ The monetized-stream holds the following information:
   "streamId": "theoads-demo",
   "description": "SGAI THEOads Demo",
   "networkCode": "89053",
-  "adURITemplate": "theo-ads://dai.break/?d=[duration]"
+  "assetURI": "https://cdn.theoplayer.com/video/big_buck_bunny/big_buck_bunny.m3u8",
+  "backdropURI": "https://demo.theoads.live/img/THEOads_double_box.svg"
```

Comment:
Do we want explicit URI's in our documentation? Don't we want to have dummy values or something like e.g. `YOUR_ASSET_URI`